packages/react-native-icons/convert.js

```diff
@@ -132,12 +132,16 @@ function processFolder(srcPath, destPath, resizable) {
       var iconContent = fs.readFileSync(srcFile, { encoding: "utf8" })
       
       var jsxCode = resizable ? svgr.default.sync(iconContent, svgrOpts, { filePath: file }) : svgr.default.sync(iconContent, svgrOptsSizedIcons, { filePath: file })
+      var rnRegex = new RegExp('(<(|\/))(svg|path|rect|g)', 'g')
```

Comment:
As majority of our icons contains only those tags I decided to use regex to change casing